### DISCUSSION TITLE
vscode devcontainer cannot build image and run container on docker engine for linux in wsl

### DISCUSSION THREAD

**Discussion by xyuxy4** [devcontainers/community#163] [Q&A]
I am using native docker engine cli in wsl distro without having docker desktop installed on windows.When devcontainer trying to follow the json file to build the container,it seems it's looking for docker installed on windows with wsl2 integrated instead of linux ones therefore it fails to finish building.
![image](https://github.com/user-attachments/assets/ee618125-1b69-4f67-a710-dd5130f259b3)

**Comment by bamurtaugh** [comment]
Thanks for opening. If you open a terminal in VS Code while connected to WSL, what does `docker info` return?
